### ISSUE TITLE
Pasting magnet link in URL does search instead of loading web torrent

### ISSUE THREAD

**Issue by srirambv** [brave/brave-browser#1435]
## Description 

While testing #857, found that pasting the magnet link in URL does a search instead of loading webtorrent

## Steps to Reproduce

   1. Install 0.55.11
   2. Paste this link `magnet:?xt=urn:btih:dd8255ecdc7ca55fb0bbf81323d87062db1f6d1c&dn=Big+Buck+Bunny` in URL and press enter
   3. Loads a search result instead of loading webtorrent

## Actual result:

![443](https://user-images.githubusercontent.com/17010094/46509962-f1b67300-c813-11e8-90fa-68c73e330f77.gif)

## Expected result:
Should load webtorrent 

## Reproduces how often: 

Easy on Windows and Linux

## Brave version (chrome://version info)

Brave | 0.55.11 Chromium: 70.0.3538.35 (Official Build) beta (64-bit)  
-- | -- 
Revision | 28dcb499844fa40c28d5f62e337876cb936f79f5-refs/branch-heads/3538@{#678}
OS | Linux & Windows

### Reproducible on current release:
- Does it reproduce on brave-browser dev/beta builds? Yes issue on both Beta and Dev
- Does it reproduce on browser-laptop? NA

### Website problems only:  
- Does the issue resolve itself when disabling Brave Shields? 
- Is the issue reproducible on the latest version of Chrome? 

### Additional Information

Issue only reproduces on Windows and Linux not on macOS

**Comment by LaurenWags** [comment]
Reproduced on macOS using 

Brave | 0.55.11 Chromium: 70.0.3538.35 (Official Build) beta(64-bit)
-- | --
Revision | 28dcb499844fa40c28d5f62e337876cb936f79f5-refs/branch-heads/3538@{#678}
OS | Mac OS X

**Comment by kjozwiak** [comment]
Can be verified once https://github.com/brave/brave-browser/issues/2479 has been fixed/addressed.

**Comment by srirambv** [comment]
Blocked on #2479

**Comment by GeetaSarvadnya** [comment]
Verification passed on 


Brave | 0.59.6 Chromium: 71.0.3578.80 (Official Build) beta (64-bit)
-- | --
Revision | 2ac50e7249fbd55e6f517a28131605c9fb9fe897-refs/branch-heads/3578@{#860}
OS | Windows

- Verified the link `magnet:?xt=urn:btih:dd8255ecdc7ca55fb0bbf81323d87062db1f6d1c&dn=Big+Buck+Bunny` loads the Webtorrent page
![image](https://user-images.githubusercontent.com/38657976/49937715-aac4b800-fefd-11e8-9a89-c7798759a397.png)

**Comment by LaurenWags** [comment]
@yrliou could you address if pasting the magnet link into a Tor window should perform a search (as is current behavior) or if it should do something else? We can get a new issue logged if current behavior is not expected.
